src/Illuminate/Support/ServiceProvider.php

```diff
@@ -134,6 +157,78 @@ protected function publishes(array $paths, $group = null)
     }
 
     /**
+     * Register a view path to be published to the app's configured view directory.
+     *
+     * @param $path
+     * @param $namespace
+     * @param null $group
+     */
+    protected function publishesViews($path, $namespace, $group = null)
+    {
+        $viewPaths = $this->app['config']->get('view.paths', [resource_path('views/vendor/')]);
+        $this->publishes([
+            $path => rtrim($viewPaths[0], '/')."/vendor/$namespace",
```

Comment:
If someone calls `publishesViews` twice with the same namespace, it will simply add both paths to the list of of paths to publish. If those paths contain files with the same name, it could cause some unexpected behavior (it would act differently depending on the `--force` flag when `vendor:publish` is called), but I think that's enough of an edge case that we don't need to worry about it.

If they call `publishesViews` twice with different namespaces, it will work as expected.

If the user has two or more paths in `config('view.paths')`, then all vendor views will be published to the first path in the array. So, for example, if you view config was:

```php
return [
    // ...
    'paths' => [
        resource_path('views'),
        base_path('secondary_views')
    ]
];
```

…then calling `vendor:publish` would publish views to `./resources/views` and leave `./secondary_views` alone.

I think that behavior makes the most sense. Is there something that I'm missing?